ci-operator/step-registry/openshift/e2e/openstack/nfv/openshift-e2e-openstack-nfv-workflow.yaml

```diff
@@ -0,0 +1,19 @@
+workflow:
+  as: openshift-e2e-openstack-nfv
+  steps:
+    pre:
+      - chain: ipi-openstack-pre
+    test:
+      - ref: openshift-e2e-test
+    post:
+      - chain: ipi-openstack-post
+    env:
+      BASE_DOMAIN: shiftstack.devcluster.openshift.com
+      CONFIG_TYPE: "minimal"
+      TEST_SKIPS: |
```

Comment:
```
Error: no tests were found matching the TEST_SKIPS regex:
 The HAProxy router should be able to connect to a service that is
idled because a GET on the route will unidle it 
```
I think a line break here is making an invalid regex

Reply:
fixed, thanks